Dockerfile

```diff
@@ -1,28 +1,26 @@
-FROM node:6.9.4-slim
+FROM centos:7
+
+RUN yum install -y epel-release libappindicator
+ADD ./google-x86_64.repo /etc/yum.repos.d/external.repo
+RUN yum install -y nodejs google-chrome-stable
+RUN yum update -y
+
 WORKDIR /tmp
 COPY webdriver-versions.js ./
-ENV CHROME_PACKAGE="google-chrome-stable_59.0.3071.115-1_amd64.deb" NODE_PATH=/usr/local/lib/node_modules:/protractor/node_modules
-RUN npm install -g protractor@4.0.14 minimist@1.2.0 && \
-    node ./webdriver-versions.js --chromedriver 2.32 && \
+ENV NODE_PATH=/usr/lib/node_modules:/protractor/node_modules
+RUN npm install -g protractor minimist
```

Comment:
Good question. Will try to rebuild with latest to see the impact :)

Reply:
Yeah we have to pin it. Its because of older version of Chrome:
![download 3](https://user-images.githubusercontent.com/1668218/39427618-c1591100-4c84-11e8-8ebf-e1da8b70c34c.png)